### PR TITLE
Update Makefile

### PR DESCRIPTION
**Pull request by apanvalkar**
DPSUST-4664 - added prune on start, update and force update

@KJMascarenhas

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -93,9 +93,10 @@ distclean: clean
 
 # update docker services
 update: remove-old-images opt dpx-apigateway.env
-	. ./dpx-container-tags && . ./svc.env && export FLUENTD_CONFIG_DIGEST=$(shell date -r ./config/fluent.conf +%s) && export START_DATE=$(shell date --iso-8601=seconds) && $(DOCKER) stack deploy --prune -c dpx.yml dpx --with-registry-auth
+	. ./dpx-container-tags && . ./svc.env && export FLUENTD_CONFIG_DIGEST=$(shell date -r ./config/fluent.conf +%s) && export START_DATE=$(shell date --iso-8601=seconds) && $(DOCKER) system prune -a -f && $(DOCKER) stack deploy --prune -c dpx.yml dpx --with-registry-auth
 
 force-update: remove-old-images opt dpx-apigateway.env
+	$(DOCKER) system prune -a -f
```
Comment (KJMascarenhas) [inline Makefile]: @apanvalkar should this action be done after the stack has been shutdown - so following the next line which does a wait for all services to exit?

**Review by KJMascarenhas — COMMENTED** [review]
Other than the one comment remaining changes look good. I guess this was specifically for addressing the cleanup via prune and not for overall Makefile improvements requested in a different Jira issue.

File: Makefile (hunk 1 above)
Comment (apanvalkar) [reply Makefile]: @KJMascarenhas  good catch.

Please re-review and merge if all good.

Comment (apanvalkar) [reply Makefile]: I will re-rest once pused to develop on our dev vm.

**Review by KJMascarenhas — APPROVED** [review]

**Comment by KJMascarenhas** [comment]
Resolving merge request with rebase & merge

**Pull request merged**